src/app/shared/notification/notification.component.html

```diff
@@ -24,6 +24,16 @@ <h3>{{message.summary}}</h3>
   </button>
 </ng-template>
 
-
-
-
+<ng-template #problemToastContent let-message="message">
+  <h3>{{message.summary}}</h3>
+  <p id="message-details">{{message.detail}}</p>
+  <div id="message-errors" *ngFor="let error of message.data.errors">
+    <p>{{error}}</p>
+  </div>
+  <a pButton class="p-button-secondary"
+    *ngIf="message.data.type !== '' && message.data.type !== undefined && message.data.type !== 'about:blank'"
```

Comment:
nit: What if the `type` is `null`? :) A regular "truthyness" check ought to be enough

```suggestion
    *ngIf="message.data.type && message.data.type !== 'about:blank'"
```

Reply:
Nice!
Just me not being JS-idiomatic from working in Go haha